<ui version="4.0" >
 <class>CSimulatedMultiTracerGuiComp</class>
 <widget class="QWidget" name="CSimulatedMultiTracerGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Encoder</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="1" >
      <widget class="QDial" name="EncoderDial" >
       <property name="maximum" >
        <number>4095</number>
       </property>
       <property name="pageStep" >
        <number>256</number>
       </property>
       <property name="wrapping" >
        <bool>true</bool>
       </property>
       <property name="notchTarget" >
        <double>128.000000000000000</double>
       </property>
       <property name="notchesVisible" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QSpinBox" name="EncoderSB" >
       <property name="maximum" >
        <number>4095</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="BitsTable" >
     <property name="gridStyle" >
      <enum>Qt::DashLine</enum>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>false</bool>
     </property>
     <row>
      <property name="text" >
       <string>0</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>15</string>
      </property>
     </row>
     <column>
      <property name="text" >
       <string>Inputs</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Outputs</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Output Counter</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>EncoderDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EncoderSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel" >
     <x>232</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EncoderSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EncoderDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>232</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel" >
     <x>124</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
